apps/dashboard/src/components/workflow-editor/steps/email/blocks/cards.tsx

```diff
@@ -0,0 +1,347 @@
+    description: 'Card: Horizontal information card with CTA',
+    searchTerms: ['logo', 'text', 'image', 'horizontal', 'card'],
+    preview: '/images/email-editor/horizontal-card-with-image-preview.webp',
+    icon: <HorizontalCardWithImage className="size-4" />,
+    command: ({ editor, range }) => {
+      track(TelemetryEvent.EMAIL_BLOCK_ADDED, {
+        type: 'header',
+      });
+
+      editor
+        .chain()
+        .deleteRange(range)
+        .insertContent({
+          type: 'columns',
+          attrs: { showIfKey: null, gap: 8 },
+          content: [
+            {
+              type: 'column',
+              attrs: {
+                width: 'auto',
+              },
+              content: [
+                {
+                  type: 'image',
+                  attrs: {
+                    src: 'https://prod-novu-app-bucket.s3.us-east-1.amazonaws.com/assets/email-editor/header-hero-image.webp',
+                    alt: null,
+                    title: null,
+                    alignment: 'center',
+                    externalLink: null,
+                    isExternalLinkVariable: false,
+                    isSrcVariable: false,
+                    showIfKey: null,
+                    height: '204',
+                    width: '282',
+                    borderRadius: 8,
+                    lockAspectRatio: false,
+                  },
+                },
+              ],
+            },
+            {
+              type: 'column',
+              attrs: {
+                width: 'auto',
+              },
+              content: [
+                {
+                  type: 'paragraph',
+                  attrs: { textAlign: null, showIfKey: null },
+                  content: [
+                    {
+                      type: 'text',
+                      marks: [{ type: 'bold' }],
+                      text: 'Multi-Environment support',
+                    },
+                  ],
+                },
+                { type: 'spacer', attrs: { height: 8, showIfKey: null } },
+                {
+                  type: 'paragraph',
+                  attrs: { textAlign: null, showIfKey: null },
+                  content: [
+                    {
+                      type: 'text',
+                      text: "Novu's Multi-Environment Support introduces a structured, secure, and efficient way to handle your notification workflows at every stage.",
+                    },
+                  ],
+                },
+                {
+                  type: 'button',
+                  attrs: {
+                    text: 'Learn more',
+                    isTextVariable: false,
+                    isUrlVariable: false,
+                    alignment: 'right',
+                    variant: 'filled',
+                    borderRadius: 'smooth',
+                    buttonColor: '#f8f8f8',
+                    textColor: '#141313',
+                    showIfKey: null,
```

Comment:
please make the button smaller and maybe add some margin to move it to the bottom, but margin is not supported so you would have to update it in the maily pr: 
```
paddingBottom: 6,
paddingLeft: 24,
paddingRight: 24,
paddingTop: 6,
marginTop: 10,
```

Reply:
please adjust all the buttons